config/templates/database.template.yml

```diff
@@ -1,10 +1,14 @@
-development:
+shared:
```

Comment:
With the addition of this `shared` section, do you need `adapter`, `pool`, and `timeout` in the development and test sections?

Reply:
The shared sections aren't working the way I expected them to, but I'd like to come back to it in a different commit.